lib/mongoid/collection_configurable.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Mongoid
+
+  # Encapsulates behavior around defining collections.
+  module CollectionConfigurable
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def create_collection
+        if coll_options = collection.database.list_collections(filter: { name: collection_name.to_s }).first
```

Comment:
The fact that this method (and thus rake task) doesn't create collections that don't have collection options specified needs to be documented. I think it's better to create all collections always? @Neilshweky

Reply:
I like the idea of creating all collections regardless of `store_in` macro, it seems to be more intuitive.

Reply:
I updated the code after I got @p-mongo comment, now it always creates collections.